linux/LinuxProcess.c

```diff
@@ -362,6 +363,17 @@ static void LinuxProcess_rowWriteField(const Row* super, RichString* str, Proces
          xSnprintf(buffer, n, "N/A  ");
       }
       break;
+   case SYSCALL: {
+      const char* syscall;
+      if (lp->syscall) {
+         syscall = lp->syscall;
```

Comment:
Would it make sense to translate the number into a name, e.g. via by creating a lookup table at compile time, see https://stackoverflow.com/a/23249568?
Also maybe handle the special value `-1` (*otherwise blocked*)?

Reply:
Firstly, I agree with your opinion, using a lookup table is correct, but maintaining lookup tables for different architectures can be cumbersome (the x64 platform also needs to be backward compatible with x86), so this submission is not prepared. If anyone is interested, they can work for it, otherwise when I have spare time, I'll take care of it myself.